code/modules/fallout/turf/ground.dm

```diff
@@ -147,7 +148,7 @@
 
 /turf/open/indestructible/ground/outside/desert/Initialize()
 	. = ..()
-	if(prob(1))
+	if(prob(2))
```

Comment:
This doubles the chance of finding stuff with a metal detector correct?

Reply:
Should not double the chance, but increase it by one percent (from one percent), if my knowledge of how the prob proc works. 

From the guide:
>prob(P)
>returns with probability P percent